concourse/steps/update_component_deps.py

```diff
@@ -377,6 +378,125 @@ def _import_release_notes(
     return release_notes
 
 
+def format_bom_diff(component_diff, include_internal_links=False):
+    changed_components = []
+    added_components = []
+    removed_components = []
+    changed_versions = []
+
+    if (hasattr(component_diff, 'cidentities_only_left') and
+        hasattr(component_diff, 'cidentities_only_right')):
```

Comment:
nit: I prefer following formatting (due to readability)
```python
if (
  hasattr(...)
  and hasattr(...)
):
  pass
```

Reply:
however, IIRC, component-diff is a dataclass - what's the point of checking for attributes to exist (they should always be present)?